### PR TITLE
added react-app

### PR DESCRIPTION
**Pull request by Ganji-NaveenKumar**
Adding react-app folder into master

### CODE REVIEW

File: my-app/src/assets/Vector.png
Comment (saicharan-zemoso) [inline my-app/src/assets/Vector.png:1]: import svgs and put them in public/assets/icons  folder

File: my-app/src/components/atoms/Typography/index.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { Typography } from "@mui/material";
```
Comment (saicharan-zemoso) [inline my-app/src/components/atoms/Typography/index.tsx]:
```suggestion
import { Typography as MuiTypography } from "@mui/material";
```

File: my-app/src/components/atoms/Typography/index.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+interface TypographyProps {
+  children: React.ReactNode;
+}
+const TypographyComp: React.FC<TypographyProps> = ({ children }) => {
+  return <Typography>{children}</Typography>;
+};
```
Comment (saicharan-zemoso) [inline my-app/src/components/atoms/Typography/index.tsx]: rename it to Typography

File: my-app/src/components/atoms/Icon/index.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  height: string;
+}
+const StyledIcon = styled.img`
+  width: width;
+  height: height;
+`;
+const IconComp: React.FC<Iconprops> = ({ src, alt }) => {
+  return <StyledIcon src={src} alt={alt} />;
+};
+export default IconComp;
```
Comment (saicharan-zemoso) [inline my-app/src/components/atoms/Icon/index.tsx]: rename it to Icon

File: my-app/src/components/molecules/Card/data.tsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+
+import image from '../../../assets/Vector.png';
+import image1 from '../../../assets/bike.png';
+import image2 from '../../../assets/train.png';
+import image3 from '../../../assets/car.png';
```
Comment (saicharan-zemoso) [inline my-app/src/components/molecules/Card/data.tsx]: should not import png as module

File: my-app/src/components/molecules/Card/index.tsx [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+      </Stack>
+      <Stack>
+        <StyledTypo>
+          <div id="text">
+            <TypographyComp>User Experience Designer</TypographyComp>
+            <TypographyComp>HP</TypographyComp>
+            <TypographyComp>Hyderabad, Telangana, India</TypographyComp>
+          </div>
+        </StyledTypo>
+      </Stack>
```
Comment (saicharan-zemoso) [inline my-app/src/components/molecules/Card/index.tsx]: remove div tags and use mui components

File: my-app/src/components/molecules/IconTypography/index.tsx [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+          {iconsData.map((icon, index) => (
+            <IconComp
+              key={index}
+              src={icon.src}
+              alt={icon.alt}
+              width="24px"
+              height="24px"
+            />
+          ))}
+        </div>
```
Comment (saicharan-zemoso) [inline my-app/src/components/molecules/IconTypography/index.tsx]: same here

File: my-app/src/components/molecules/Card/index.tsx [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+        </StyledIcon>
+      </Stack>
+      <Stack>
+        <StyledTypo>
+          <div id="text">
+            <TypographyComp>User Experience Designer</TypographyComp>
+            <TypographyComp>HP</TypographyComp>
+            <TypographyComp>Hyderabad, Telangana, India</TypographyComp>
+          </div>
+        </StyledTypo>
```
Comment (saicharan-zemoso) [inline my-app/src/components/molecules/Card/index.tsx]: divide these in to smaller pieces and the whole component is an organism

**Review by saicharan-zemoso — CHANGES_REQUESTED** [review]
resolve all changes

File: my-app/src/assets/Vector.png
Comment (Ganji-NaveenKumar) [reply my-app/src/assets/Vector.png:1]: changes modified

File: my-app/src/components/atoms/Typography/index.tsx (hunk 1 above)
Comment (Ganji-NaveenKumar) [reply my-app/src/components/atoms/Typography/index.tsx]: updated

File: my-app/src/components/atoms/Typography/index.tsx (hunk 2 above)
Comment (Ganji-NaveenKumar) [reply my-app/src/components/atoms/Typography/index.tsx]: updated

File: my-app/src/components/atoms/Icon/index.tsx (hunk 3 above)
Comment (Ganji-NaveenKumar) [reply my-app/src/components/atoms/Icon/index.tsx]: updated

File: my-app/src/components/molecules/Card/data.tsx (hunk 4 above)
Comment (Ganji-NaveenKumar) [reply my-app/src/components/molecules/Card/data.tsx]: changes modified

File: my-app/src/components/molecules/Card/index.tsx (hunk 5 above)
Comment (Ganji-NaveenKumar) [reply my-app/src/components/molecules/Card/index.tsx]: removed all divs and added mui

File: my-app/src/components/molecules/IconTypography/index.tsx (hunk 6 above)
Comment (Ganji-NaveenKumar) [reply my-app/src/components/molecules/IconTypography/index.tsx]: modified

File: my-app/src/components/molecules/Card/index.tsx (hunk 7 above)
Comment (Ganji-NaveenKumar) [reply my-app/src/components/molecules/Card/index.tsx]: created organism

File: my-app/src/components/molecules/TypographyBody/index.tsx [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+const StyledTypography = styled.div`
+  text-align: start;
+`;
+const TypographyBody: React.FC = () => {
+  return (
+    <StyledTypography>
+      <Typography>User Experience Designer</Typography>
+      <Typography>HP</Typography>
+      <Typography>Hyderabad, Telangana, India</Typography>
+    </StyledTypography>
```
Comment (saicharan-zemoso) [inline my-app/src/components/molecules/TypographyBody/index.tsx]: how are  the typography's styled ?

File: my-app/src/utils/constants/data.tsx [hunk 9]
```diff
@@ -0,0 +1,8 @@
+
+import ImagePath from ".";
+export const iconsData = [
+  { src: ImagePath.image, alt: 'Image not found' },
+  { src: ImagePath.image1, alt: 'Image not found' },
+  { src: ImagePath.image2, alt: 'Image not found' },
+  { src: ImagePath.image3, alt: 'Image not found' }
+];
```
Comment (saicharan-zemoso) [inline my-app/src/utils/constants/data.tsx]: place them in the constants file no need of separate data file

**Review by saicharan-zemoso — CHANGES_REQUESTED** [review]
Please provide the pic of the component  that was done

File: my-app/src/components/molecules/TypographyBody/index.tsx (hunk 8 above)
Comment (Ganji-NaveenKumar) [reply my-app/src/components/molecules/TypographyBody/index.tsx]: modified

File: my-app/src/utils/constants/data.tsx (hunk 9 above)
Comment (Ganji-NaveenKumar) [reply my-app/src/utils/constants/data.tsx]: changed

**Comment by Ganji-NaveenKumar** [comment]
> Please provide the pic of the component that was done

add in public folder as component.png

**Review by saicharan-zemoso — CHANGES_REQUESTED** [review]
add font, colors, and spacing, and make the component same as given in the figma.
and after making the changes  add a new screenshot in the comments

**Comment by Ganji-NaveenKumar** [comment]
Screenshot 2024-07-19 124744

**Review by saicharan-zemoso — CHANGES_REQUESTED** [review]
check if the font family is getting applied and check the colors and spacing

File: my-app/src/components/molecules/IconHeader/index.tsx [hunk 10]
```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import Icon from "../../atoms/Icon";
+import styled from "@emotion/styled";
+import ImagePath from "../../../utils/constants";
+const ImageComp = styled.div`
```
Comment (saicharan-zemoso) [inline my-app/src/components/molecules/IconHeader/index.tsx]: don't use div. use mui component like box

**Review by saicharan-zemoso — CHANGES_REQUESTED** [review]
please provide the screenshot of the updated component every time it is updated.

**Comment by Ganji-NaveenKumar** [comment]
image

**Review by saicharan-zemoso — APPROVED** [review]
looks good to me

**Pull request merged**